stdlib/LinearAlgebra/src/uniformscaling.jl

```diff
@@ -227,6 +227,9 @@ Broadcast.broadcasted(::typeof(*), J::UniformScaling,x::Number) = UniformScaling
 
 Broadcast.broadcasted(::typeof(/), J::UniformScaling,x::Number) = UniformScaling(J.λ/x)
 
+(^)(A::UniformScaling, x::Number) = UniformScaling((A.λ)^x)
+Broadcast.broadcasted(::typeof(^), J::UniformScaling,x::Number) = UniformScaling(J.λ^x)
```

Comment:
```suggestion
Broadcast.broadcasted(::typeof(^), J::UniformScaling, x::Number) = UniformScaling(J.λ^x)
```
No need to preserve the "klemp".